pacman/index.html

```diff
@@ -212,10 +212,6 @@ <h1>Powered by Amplitude</h1>
 								<div class='email-form-row'>
 									<div id='form-validater'></div>
 								</div>
-								<div class='email-form-row'>
-									<div>Email:</div>
-									<input type='text' id='playerEmail' autofocus/>
-								</div>
 								<div class='email-form-row'>
 									<div>Name:</div>
 									<input type='text' id='playerName'/>
```

Comment:
Do we want to add a note something like "We don't want to collect any personal info, so we use player name as unique id, please be creative in your playerName." to reminder people of the unique id things. So that the people on site doesn't need to always do that and people will not confused by the duplicate names.

Reply:
Uh this doesn't fit in the UI/is way too long

Reply:
would't that cause people's score or other charts mixed together?

Reply:
Each score is a separate row in the leader board => 2 scores

Charts are usually aggregate (and hopefully based on "totals") - we can verify tomorrow

user activity is probably fine (they can see its the name they put in showing new events up)

Reply:
changing to "Player Name"